lighthouse-treemap/app/src/main.js

```diff
@@ -15,6 +15,9 @@ const DUPLICATED_MODULES_IGNORE_THRESHOLD = 1024;
 const DUPLICATED_MODULES_IGNORE_ROOT_RATIO = 0.01;
 
 const logEl = document.querySelector('#lh-log');
+if (!logEl) {
```

Comment:
this is the same check as [in `lighthouse-viewer`](https://github.com/GoogleChrome/lighthouse/blob/9827fdbfd939e6de2a0017390c3a53d3b52a5a6c/lighthouse-viewer/app/src/main.js#L12-L15), but happy to do whatever